app/javascript/src/components/editor/CodeMirror.svelte

```diff
@@ -174,17 +174,17 @@
   }
 
   function insertSemicolon(view, transaction) {
+    if (!$settings["autocomplete-semicolon"]) return
```

Comment:
Wouldn't it make more sense to just not call the function?

```ts
  function autocompleteFormatting(view, transaction) {
    if (transaction.transactions.every(tr => ["input.complete"].includes(tr.annotation(Transaction.userEvent)))) {
      indentMultilineInserts(view, transaction)
      if ($settings["autocomplete-semicolon"])
          insertSemicolon(view, transaction)
    }
  }
```

Reply:
yes, not sure why it put it the function